lib/Dancer2/Manual/Migration.pod

```diff
@@ -298,6 +300,26 @@ function with decoded input.
 
 =back
 
+=head3 Templating engine changes
+
+Whereas in Dancer1 it was acceptable to write:
+
+    template '/foo/bar'
+
+In Dancer2, the leading slash is not permitted:
```

Comment:
@xsawyerx - looks like it's both.

D1 simple: `/index.tt` treated as `APPDIR/views/index.tt`.
D1 template_toolkit: `/index.tt` treated as `APPDIR/views/index.tt`.
D2 simple: `/index.tt` treated as `APPDIR/views/index.tt`.
D2 template_toolkit: `/index.tt` behaves differently - looks like it's looking for `/index.tt`, i.e. at the filesystem root.

I guess 'not permitted' is not the right phrase. But I think we should flag this change (assuming it's deliberate), even if it's strictly not Dancer Core but the TT2 plugin.